libraries/AP_HAL_Linux/RCOutput_AioPRU.cpp

```diff
@@ -53,8 +61,16 @@ void RCOutput_AioPRU::init()
    // Reset PRU
    *ctrl = 0;
 
-   // Load firmware
+   // Loading firmware
+   // for ARM64 using loading PRU code in block instead memcpy
+#if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_POCKET2   
```

Comment:
Please come up with a generic define for this memcpy-or-wordwise-copy, rather than using the board subtype here.  Then define it in your hwdef.dat and default it.... somewhere, maybe at the top of this file